bench.txt

```diff
@@ -1,9 +1,9 @@
-consume#consume vlq mappings x 109145 ops/sec ±2.27% (89 runs sampled)
-consume#consume flatbuffer x 129311 ops/sec ±3.39% (78 runs sampled)
-consume#consume JS Mappings x 69705 ops/sec ±2.35% (86 runs sampled)
-serialize#Save buffer x 484 ops/sec ±2.04% (79 runs sampled)
-serialize#Serialize to vlq x 240 ops/sec ±1.1% (87 runs sampled)
-modify#positive line offset x 334 ops/sec ±13.03% (36 runs sampled)
-modify#negative line offset x 200 ops/sec ±2.39% (68 runs sampled)
-modify#positive column offset x 3801396 ops/sec ±0.76% (92 runs sampled)
-modify#negative column offset x 3833932 ops/sec ±0.86% (96 runs sampled)
\ No newline at end of file
+consume#consume vlq mappings x 66617 ops/sec ±2.62% (86 runs sampled)
+consume#consume flatbuffer x 114374 ops/sec ±4% (82 runs sampled)
+consume#consume JS Mappings x 26181 ops/sec ±4.81% (78 runs sampled)
```

Comment:
Did this get slower?

Reply:
It should be faster according to the benchmark action twice as fast

Reply:
Interesting. Was the previous bench.txt out of date? Or maybe I misread it?

Reply:
Yeah idk, maybe it's not faster on OS X, getting similar results when running the benchmark again.

Maybe I shouldn't make these JSON hack changes? It was based on some benchmarks the napi-rs maintainer shared, strange that the native way of doing this is so slow.

Reply:
Yeah it is weird. Maybe because of copying the JS strings one by one? Wonder what would happen if we used buffers... In any case, it doesn't seem to make things slower at least. Fine with merging it for now.